apps/tangle-dapp/data/Payouts/usePayoutsAvailability.ts

```diff
@@ -0,0 +1,32 @@
+import { isSubstrateAddress } from '@webb-tools/dapp-types';
+import { useEffect, useMemo, useState } from 'react';
+
+import useActiveAccountAddress from '../../hooks/useActiveAccountAddress';
+import { convertToSubstrateAddress } from '../../utils';
+import usePayouts from '../DelegationsPayouts/usePayouts';
+
+const usePayoutsAvailability = () => {
+  const activeAccountAddress = useActiveAccountAddress();
+
+  const substrateAddress = useMemo(() => {
+    if (!activeAccountAddress) return '';
+
+    if (isSubstrateAddress(activeAccountAddress)) return activeAccountAddress;
+
+    return convertToSubstrateAddress(activeAccountAddress);
+  }, [activeAccountAddress]);
```

Comment:
You may want to use the existing [`useSubstrateAddress`](https://github.com/webb-tools/webb-dapp/blob/develop/apps/tangle-dapp/hooks/useSubstrateAddress.ts#L16) hook that already abstracts this for you.